pty/pty.go

```diff
@@ -34,14 +53,6 @@ type PTY interface {
 	//
 	// The same stream would be used to provide user input: pty.Input().Write(...)
 	Input() ReadWriter
```

Comment:
It feels odd to have `Input` which is a `ReadWriter` but also have `InputWriter` on the `PTYCmd` interface. I'm not sure what better naming would be, just pointing out that it's not entirely obvious to me as a reader which to use.

Reply:
Perhaps pty.PTY and pty.Command should be two completely separate types?

Then command can also have Input() which is io.Writer and there’s no confusion between the two.

Reply:
I think that makes sense!